.github/workflows/Release.yml

```diff
@@ -0,0 +1,117 @@
+name: GitHub Release with APKs
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 45
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2    
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build all projects
+        run: ./scripts/gradlew_recursive.sh assembleDebug
+
+      - name: Create Release
+        id: create_release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: "${{ github.ref }}"
```

Comment:
Since you just uncommented this, can you reenable the build again to see how this looks like (and if it works)

Reply:
It won't work, the ref will be a commit id here. Only for the tag will the ref be correct.

Reply:
...and this definitely works. It's taken from the official action docs: https://github.com/actions/create-release